### DISCUSSION TITLE
Adding custom emojis to retext-emoji?

### DISCUSSION THREAD

**Discussion by OwenMelbz** [retextjs/retext#93] [Q&A] [answered]
Hi,

We currently use emoji-mart on our website as an emoji picker which supports custom emojis e.g. `:gg: or :wtf:` but when rendering the markdown using unified/retext/etc it skips these as not real shortcuts for emojis.

How would we go about supporting these?

Thanks

**Comment by wooorm** [comment, marked as answer]
hi!

I’m not sure how that would work. This project doesn’t do that.

I also don’t see how this project (retext-emoji) is used with remark? Do you mean [`remarkjs/remark-gemoji`](https://github.com/remarkjs/remark-gemoji)?

**Reply by OwenMelbz** [reply]
Hi, sorry I thought this project converts the shortcuts into emojis e.g. `:fire: > 🔥` which is why I thought it would be the place to convert custom shortcuts into custom emojis.

**Reply by wooorm** [reply]
It does that, but in retext, which can’t yet affect remark, see for more info: https://github.com/remarkjs/remark-retext/tree/main#when-should-i-use-this.

Are you using markdown? remark? Or not? Are you using this project already? Or something else?

**Comment by elidakirigo** [comment]
thats awesome
